src/test/java/com/training/sample/tests/SortingFilter.java

```diff
@@ -0,0 +1,65 @@
+package com.training.sample.tests;
```

Comment:
Why is the sorting logic inside tests package?
All the implementation code should go in main package.
Also use proper indentation for the code.